Updates test coverage

p-galligan opened the pull request:
Updates test coverage on routines and helpers.

Fixes #56

Commit:
Updates test coverage

Updates test coverage on routines and helpers

helrond commented on process_request/tests.py:
```diff
@@ -197,6 +197,12 @@ def test_get_size(self):
             instance = json_from_fixture(fixture)
             self.assertEqual(get_size(instance), size)
 
+        instance = json_from_fixture("instances_error.json")
+        error_message = "Error parsing instances"
+        with self.assertRaises(Exception) as e:
+            get_size(instance)
+        self.assertEqual(str(e.exception), error_message)
+
```
I'm not sure it's necessary to test the exception message here, since it's hardcoded in the method and we're just testing that our two hardcoded strings match. I'd be inclined to just do away with lines 200 and 204.

p-galligan replied:
I assume you mean line 201, but yeah, easy enough to change.

helrond commented on process_request/tests.py:
```diff
@@ -267,6 +278,17 @@ def test_get_data(self):
+    @aspace_vcr.use_cassette("aspace_request.json")
+    @patch("asnake.client.web_client.ASnakeClient.get")
+    def test_invalid_get_data(self, mock_as_get):
+        error_message = "This is an error!"
```
I know I did this but I think this should be a random string rather than a hardcoded one.

helrond commented on process_request/tests.py:
```diff
@@ -280,8 +302,8 @@ def test_send_aeon_requests(self, mock_get_data):
         request_type = "foo"
-        with self.assertRaises(Exception, msg="Unknown request type '{}', expected either 'readingroom' or 'duplication'".format(request_type)):
-            AeonRequester().get_request_data(request_type, **data)
+        with self.assertRaises(ValueError, msg="Unknown request type '{}', expected either 'readingroom' or 'duplication'".format(request_type)):
+            AeonRequester().get_request_data(request_type, "https://dimes.rockarch.org", **data)
```
This is great - is there a chance we can use the same kind of detail in assertRaises for other exceptions where we're testing the message (like line 290 above)?

p-galligan replied:
I _think_ I've gotten what you were asking, but let me know if I missed the mark.

helrond reviewed (changes requested):
Looking good - a couple of minor tweaks.

Commit:
minor test tweaks

Some changes to `assertRaises` in tests to check that specific error messages are being raised.
